blog_guidelines.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: Blog Guidelines
+tagline: Guidelines for Submitting a Post to the Jupyter Blog.
+permalink: /blog_guidelines
+---
+
+## Content Guidelines
+The Project Jupyter blog is designed for community members to share valuable and educational content about Jupyter. Blog posts can be about various topics, for example how you got started with Jupyter, the latest extension you built around Jupyter, a quick tutorial on how to use a specific feature, and more. This channel is powered by our community members so there are no particular requirements for how the blog post has to be written. 
+
+Here is an example of a potential structure:
+
+- Title
+- Subtitle (max 140 characters) – summary of what the blog post is about
+- Provide one image that represents your project - it can also be a code snippet
+- Introduce your topic
+- Why is this topic important? What are the benefits for the Jupyter community?
+- Do you have an interesting user story or usage example? Include this here.
+- Pick a specific usage example and provide a step-by-step guide so the readers can follow along. Make sure to include visuals or
+  code examples so readers can follow along easily.
+- Thank/acknowledge the people who helped make the work described in the blog post possible. This includes anyone who helped you
+  write the blog post itself.
+- End the article with the forward-looking statement about your project or research
+
+Blog Post Examples:
+- Shorter post: [European Commission Funds Jupyter Bug Bounty Program](https://blog.jupyter.org/european-commission-funds-jupyter-bug-bounty-program-b7b96f9831e6)
+- Longer post: [Generative AI in Jupyter](https://blog.jupyter.org/generative-ai-in-jupyter-3f7174824862)
+- Post using the Jupyter Voice: [JupyterLab is Ready for Users](https://blog.jupyter.org/jupyterlab-is-ready-for-users-5a6f039b8906)
+
+## Submission Process
+- *Create a Medium account*. If you are new to Medium, create an account by following these [instructions](https://help.medium.com/hc/en-us/articles/115004915268-Sign-in-or-sign-up-to-Medium).
+- *Request permissions*. Email the Jupyter Media Strategy working group (JMS)  at jupyter-media-strategy at googlegroups dot com  to have your Medium account to be added as the writer of the blog. Once you are added as the authorized writer, you should receive an email that allows you to upload your blog. 
```

Comment:
That won't work in the long run, I believe we are already at the maximum number of writer.

Reply:
As far as I can tell, there is no limit on the number of writers.  The number of *editors* will be reduced as part of this effort.  If we do hit the limit on the number of writers, we'll have to start removing inactive writers, who can also be re-added through the submission process.